telegram/ext/jobqueue.py

```diff
@@ -21,10 +21,26 @@
 import logging
 import time
 import warnings
+import datetime
+from numbers import Number
 from threading import Thread, Lock, Event
 from queue import PriorityQueue, Empty
 
 
+class Days(object):
+    mon = 0
+    tue = 1
+    wed = 2
+    thu = 3
+    fri = 4
+    sat = 5
+    sun = 6
+
+
+def get_days():
```

Comment:
I know what this is supposed to do, but it's still unreadable. I think the best solution would be to do something like we did in [`MessageEntity`](https://github.com/python-telegram-bot/python-telegram-bot/blob/master/telegram/messageentity.py#L83). Other ideas?